### ISSUE TITLE
Links to Rules documentation

### ISSUE THREAD

**Issue by Graham-72** [backdrop-contrib/rules#84]
There are some inbuilt links to documentation in rules.module `function rules_external_help` that currently link to Drupal pages. These need to be replaced with links to `https://github.com/backdrop-contrib/rules/wiki`.

**Comment by laryn** [comment]
Agreed, I see @bugfolder has made a start in the Wiki and it would be great to get it all moved over here and those links updated.

https://github.com/backdrop-contrib/rules/blob/1.x-2.x/rules.module#L1599-L1627
